### ISSUE TITLE
why 'ACAB' is present in the repo subtitle ?

### ISSUE THREAD

**Issue by ngouy** [ruby-lint/ruby_lint#20]
## Is your feature request related to a problem? Please describe.

![image](https://user-images.githubusercontent.com/12391171/84059915-559a2300-a989-11ea-9e18-413b57a814f5.png)

I get we may want to rename rubocop gem, and that `cop` word may be connotated and/or be related to things we don't want to be part of.

Though not sure why `ACAB` acronym is present on the project description.
Does it provide a betterment?

**Comment by polytomous** [comment]
why 'concern-trolling bootlickers' is present in the repo issues? 
====
![the person who filed the issue, and the two who offered thumbs up reactions to it; conspicuously all white adjacent](https://user-images.githubusercontent.com/13786735/84195941-6754f680-aa6d-11ea-8e18-c3230ea91697.png)

It's obvious we should rename the Rubocop gem, and that cops are connotated with brutality, injustice, and violence, and are related to things that noone should want to be a part of.

Though not sure why `concern trolling bootlickers` is present on the project issue tracker.
Do they provide betterment? 

(No.)
====
acab offers a statement of truth that faithfully represents the reality of policing. 

john oliver offers a basic primer on the police that may help you fight your urge to caress fascist leather with your tongue
https://www.youtube.com/watch?v=Wf4cea5oObY

this thread of police brutality during the protests has ballooned to over 400 distinct posts
https://twitter.com/greg_doucette/status/1269830986059124736

cops deliberately target journalists. the aclu is suing about it
https://www.aclu.org/news/free-speech/police-are-attacking-journalists-at-protests-were-suing/

cops deliberately target street medics, who are trying to help soothe the pain of the cops' brutality
https://newrepublic.com/article/157985/protest-medics-targeted-police-words

cops deliberately use chemical weapons banned by the geneva convention which are also conveniently abortifacients
https://www.thenation.com/article/archive/tear-gas-abortifacient-why-wont-anti-abortion-movement-oppose-it/

cops target the disabled; up to 50% of those murdered by the police have some form of disability
https://www.bbc.com/news/stories-45739335

https://blobbynfriends.tumblr.com/image/620105407727288320

now is a perfect time for yall to consider your stance re the cops and reevaluate. 

```
ACAB
C
A
B
```

**Comment by ngouy** [comment]
U guys are fun to stalk people on internet putting their real face in the issue. I am so afraid now !! I really thought I was well hidden behind my screen 😨. (not) I have not much to hide so fine by me.
Yes there are a lot of people trolling and I get that u may think I am "one of them". Sorry, I'm not, nor is this issue.

I myself try think about this rubocop naming stuff, and that's why I was posting here, because the idea is interesting, but having ACAB in front of a repo is kinda extreme and won't encourage all kind people to go for it. We have to find balance. Get rid of cop stuff yes, but not by replacing it by "cops are bastard" lol. This is nonsense. Sadly this kind of behaviour totally discredit this very one renaming initiative and beyond that, and also hurt similar initiatives that however can be meaningfull. 

The very fact we want to say every single cops are all bastards is exactly the kind of behaviour this whole movement blacklives matters and this repo with all kind of similar initiative are fighting against: radical ideas, amalgams and wrongly treating people for what they represent in our closed mindsets instead of the different yet cool person they may be.

I agree with you that cops are doing deeply horrible things in USA, they do even worse in other part of the globe, does it gives us the right to say all cops are bastard. Not sure. Does it gives us the right to try to change the the word we are using? 💯

In short: 
- Rename rubocop: interesting idea.
- having "ACAB" in front of a solid alternative: why?
- not sure why you think I'm a troll or a facist judging by an old pp, but fun

ps: I hope your tour on the blog was instructive, EmberJS is the way 👍
